docs/providers/aws/guide/serverless.yml.md

```diff
@@ -136,8 +136,10 @@ provider:
     lambda: true # Optional, can be true (true equals 'Active'), 'Active' or 'PassThrough'
   logs:
     restApi: # Optional configuration which specifies if API Gateway logs are used. This can either be set to true to use defaults, or configured via subproperties.
-      format: 'requestId: $context.requestId' # Optional configuration which specifies the log format to use.
-      level: INFO # Optional configuration which specifies the log level to use. May be either INFO or ERROR.
+      enableAccessLogging: true # Optional configuration which enables or disables access logging. Defaults to true.
+      format: 'requestId: $context.requestId' # Optional configuration which specifies the log format to use for access logging.
+      level: INFO # Optional configuration which specifies the log level to use for execution logging. May be set to either INFO or ERROR, or set to OFF to disable.
```

Comment:
If  `OFF`is way to turn off data logging entirely. I wouldn't support such value here (it's also not supported in AWS console UI), but just introduce `dataLogging` property, which could be turned to `false` to turn off _data_ logging.

Reply:
I think executionLogging here would be a better fit, as that's what the AWS documentation refers to it as (see https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html). I agree that having a separate field is a better idea here, it's also consistent with having accessLogging to enable/disable access logging.

Reply:
Indeed, `executionLogging` is better choice in this case.